SDK version v0.3.9 errors

Getting the following error:

```
# github.com/infisical/go-sdk/packages/api/dynamic_secrets
../../../go/pkg/mod/github.com/infisical/go-sdk@v0.3.9/packages/api/dynamic_secrets/get_secret_by_name.go:21:55: request.SecretName undefined (type GetDynamicSecretByNameV1Request has no field or method SecretName)
../../../go/pkg/mod/github.com/infisical/go-sdk@v0.3.9/packages/api/dynamic_secrets/list_lease.go:22:55: request.SecretName undefined (type ListDynamicSecretLeaseV1Request has no field or method SecretName)
```

--- a/packages/api/dynamic_secrets/get_secret_by_name.go
+++ b/packages/api/dynamic_secrets/get_secret_by_name.go
@@ -18,7 +18,7 @@ func CallGetDynamicSecretByNameV1(httpClient *resty.Client, request GetDynamicSe
 		"path":            request.SecretPath,
 	})
 
-	res, err := req.Get("/v1/dynamic-secrets/" + request.SecretName)
+	res, err := req.Get("/v1/dynamic-secrets/" + request.DynamicSecretName)
 
 	if err != nil {
 		return GetDynamicSecretByNameV1Response{}, errors.NewRequestError(callGetDynamicSecretByNameV1Operation, err)

--- a/packages/api/dynamic_secrets/list_lease.go
+++ b/packages/api/dynamic_secrets/list_lease.go
@@ -19,7 +19,7 @@ func CallListDynamicSecretLeaseV1(httpClient *resty.Client, request ListDynamicS
 			"path":            request.SecretPath,
 		})
 
-	res, err := req.Get("/v1/dynamic-secrets/" + request.SecretName + "/leases")
+	res, err := req.Get("/v1/dynamic-secrets/" + request.DynamicSecretName + "/leases")
 
 	if err != nil {
 		return ListDynamicSecretLeaseV1Response{}, errors.NewRequestError(callListDynamicSecretLeaseV1Operation, err)

